assets/styles/map-size.css

```diff
@@ -0,0 +1,17 @@
+html, body {
+    height: 100%;
+}
+
+body {
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+main {
+    flex: auto;
+}
+
+#content {
+    height: 100%;
+}
```

Comment:
On dirait qu'on peut se contenter de ça

```suggestion
#content {
    calc(100vh - 200px);
}
```

J'utilise 100vh avec une marge pour que le footer s'affiche toujours à peu près sous la bordure inférieure de l'écran

Reply:
Intéressant comme rendu, mais du coup le `200px` correspond à la hauteur du footer, codée en dur ?

Reply:
Oui, environ la hauteur du footer

Il est possible de calculer "dynamiquement" la hauteur du footer avec du JS pour calculer le viewportHeight du `footer` et modifier le CSS dynamiquement (desktop vs mobile)

L'approche de coder une hauteur en dur me semble un compromis de simplicité

Juste vérifier si ça rend pas trop mal sur mobile

Reply:
Le soucis en mobile, c'est qu'on ne peut pas scroller vers le bas ou vers le haut si on ne voit que de la carte, un doigt sur la carte, ça fait un "pan" (un déplacement de la carte).

Reply:
Pour intégrer ta solution, il me faut fournir un travail supplémentaire pour que ça fonctionne lorsque la `minHeight` de la carte est activée … est-ce que ça vaut le coup ? 

<img width="745" alt="Capture d’écran 2024-07-04 à 15 41 18" src="https://github.com/MTES-MCT/dialog/assets/101107163/b72b05d9-f902-442b-bae2-5cee6a072b1b">

Reply:
Non, bon, laissons comme ça...